src/message/MessageBrief.js

```diff
@@ -48,9 +48,9 @@ export default class MessageBrief extends React.PureComponent {
       <View style={styles.message}>
         <View style={styles.messageContentWrapper}>
           <TouchableWithoutFeedback onLongPress={onLongPress}>
-            <View style={styles.childrenWrapper}>
```

Comment:
Ok, that is a good reason. I am only concerned that a ScrollView might alter the behavior of the messages in other cases. As long as you have not encountered issues, I am cool with it.